packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -1,707 +0,0 @@
-/*
- * Copyright (c) 2018, salesforce.com, inc.
- * All rights reserved.
- * SPDX-License-Identifier: MIT
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
- */
-import { DiagnosticLevel } from '@lwc/errors';
-import parse5Errors from 'parse5/lib/common/error-codes';
-import { normalizeConfig } from '../config';
-import State from '../state';
-import parse from '../parser';
-import { errorCodesToErrorOn, errorCodesToWarnOn } from '../parser/parse5Errors';
-
-const EXPECTED_LOCATION = expect.objectContaining({
-    line: expect.any(Number),
-    column: expect.any(Number),
-    start: expect.any(Number),
-    length: expect.any(Number),
-});
-
-function parseTemplate(src: string): any {
-    const config = normalizeConfig({});
-    const state = new State(config);
-
-    const res = parse(src, state);
-    return {
-        ...res,
-        state,
-    };
-}
-
-describe('parsing', () => {
-    it('simple parsing', () => {
```

Comment:
I went through `parser.spec.ts` and found a few test scenarios that are missing in the fixtures. 

I think it might be worthwhile to go through this file to make sure all the tests that were here are covered in the fixtures.

Maybe we could add a `TODO` to comb through the `parser.spec.ts` and add any missing tests to the fixtures before deleting the file in a separate PR.

Here's some examples that I found:

https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/__tests__/parser.spec.ts#L317-L344

https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/__tests__/parser.spec.ts#L365-L388

https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/__tests__/parser.spec.ts#L576-L601

Reply:
Good call! I haven't done the due diligence of checking the coverage report. 
Let me investigate this today.